docs/DeveloperGuide.md

```diff
@@ -584,10 +767,38 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
       Use case resumes at step 2.
 
 * 4a. The user sets the wrong date.
-  
+
   Use case resumes at step 2.
 
-<h3>Use case: Removing a patient's next appointment date</h3>
+**Use case: Delete a patient's appointment**
+
+**MSS**
+
+1.  User requests to list persons
+2.  CheckUp shows a list of persons
+3.  User requests to delete appointment from a specific person in the list
+4.  CheckUp deletes the appointment
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given index is invalid.
+
+    * 3a1. CheckUp shows an error message.
+
+      Use case resumes at step 2.
```

Comment:
typo here: `resumes at step 2`

Reply:
for this i followed the given delete example, where use case resumes at step 2 if an invalid index is given